### PR TITLE
Contribution process based on #219.

### PR DESCRIPTION
**Pull request by mikeal**
This is a modification of the process we use for core giving certain rights to the admin group.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,3 +1,87 @@
...
+Courtesy should always be shown to individuals submitting issues and
+pull requests to the Node.js project.
+
+Collaborators should feel free to take full responsibility for
+managing issues and pull requests they feel qualified to handle, as
+long as this is done while being mindful of these guidelines, the
+opinions of other Collaborators and guidance of the Website Admin Group.
+
+Collaborators may **close** any issue or pull request they believe is
+not relevant for the future of the Node.js project. Where this is
```
Comment (evanlucas) [inline CONTRIBUTING.md:28]: Nodejs.org?

**Comment by fhemberger** [comment]
SGTM.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,3 +1,87 @@
...
+amongst Collaborators, consensus should be sought if possible. The
+lack of consensus may indicate the need to elevate discussion to the
+Website Admin Group for resolution (see below).
+
+All bugfixes require a test case which demonstrates the defect. The
+test should *fail* before the change, and *pass* after the change.
+
+All pull requests that modify executable code should be subjected to
+continuous integration tests on the
+[project CI server](https://ci.nodejs.org/).
```
Comment (evanlucas) [inline CONTRIBUTING.md:70]: Isn't CI on travis instead of jenkins here?

Comment (mikeal) [inline CONTRIBUTING.md:70]: there's also a jenkins thing that publishes the content in master.

Comment (evanlucas) [inline CONTRIBUTING.md:70]: ah that makes sense

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,3 +1,87 @@
...
+not relevant for the future of the Node.js project. Where this is
+unclear, the issue should be left open for several days to allow for
+additional discussion. Where this does not yield input from Node.js
+Collaborators or additional evidence that the issue has relevance, the
+issue may be closed. Remember that issues can always be re-opened if
+necessary.
+
+## Accepting Modifications
+
+All modifications to the Node.js code and documentation should be
```
Comment (phillipj) [inline CONTRIBUTING.md:37]: Node.js **website** code?

File: CONTRIBUTING.md (hunk 2 above)
Comment (phillipj) [inline CONTRIBUTING.md:70]: > there's also a jenkins thing that publishes the content in master.

What is the website related stuff called?

**Pull request merged**